topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -218,7 +234,9 @@ private Image createLogoImage(String iconUrl) {
 	 * @return value or defaultValue
 	 */
 	private String ifPresent(String value, String defaultValue) {
-		return value != null && !value.trim().isEmpty() ? value : defaultValue;
+		boolean present = value != null && !value.trim().isEmpty();
+		logger.info("No value found! Falling back to default value "+defaultValue);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)